examples/src/test/scala/docs/guide/ShoppingCartClusterApp.scala

```diff
@@ -52,7 +52,7 @@ object ShoppingCartClusterApp extends App {
       ShardedDaemonProcess(system).init[ProjectionBehavior.Command](
         name = "shopping-carts",
         numberOfInstances = ShoppingCartTags.tags.size,
-        behaviorFactory = (n: Int) => ProjectionBehavior(projection(ShoppingCartTags.tags(n))),
+        behaviorFactory = i => ProjectionBehavior(projection(ShoppingCartTags.tags(i))),
```

Comment:
Dropping the type is causing a type inference compilation issue in 2.12.

https://travis-ci.com/github/akka/akka-projection/jobs/376155383#L312